skfda/misc/metrics/sdtw_fast.pyx

```diff
@@ -0,0 +1,88 @@
+"""
```

Comment:
Is it really necessary to implement this in Cython? The current version of scikit-fda does not have non-python code, and that simplifies the release process, as we don't need to offer versions for each platform. I am open to have non-python code if necessary, but in this case I think the code can be expressed in numpy without losing performance.

Reply:
Well it is not necessary, but I compared a python implementation (with @numba.jit decorators) to the actual cython which has resulted between %5 and 10% faster in terms of computational time. The computational bottleneck is the double nested loops. I can implement it fully in numpy with @jit. I'll try to optimize the memory too.